### PR TITLE
Work in PCM-9014 make it possible to query/pull back cluster ID's

### PR DESCRIPTION
**Pull request by sferich888**
@TheDiemer I don't think we want to merge this just yet, but I wanted to file this for you to review. 
Please look at the connected JIRA where the API is still being worked out. 

Searching for cluster_ids does not seem to work, but pulling back a cluster_id with a normal case search does seem to work, using the correct field.

### CODE REVIEW

**Comment by reestr** [comment]
LGTM, tested with a couple of cluster_ids which pull back the correct cases, for single case linked and many cases linked.

File: hydra/hydra.py [hunk 1]
```diff
@@ -280,24 +297,60 @@ def put_owner(self, case_number, user=""):
...
+            if len(tags) == 1:
+                query_params.update({'q': 'case_tags:{}'.format(tags[0])})
+            else:
+                query_params.update({'q':
+                    'case_tags:({})'.format(' OR '.join(tags))})
+
+        if cluster_ids:
+            if len(cluster_ids) == 1:
+                query_params.update({'q':
+                    'case_openshift_cluster_id:{}'.format(cluster_ids[0])})
```
Comment (wking) [inline hydra/hydra.py]: No need for this case, you can just `join` for any len >= 1:

```
>>> '-'.join(['a'])
'a'
>>> '-'.join(['a', 'b'])
'a-b'
```

Comment (sferich888) [reply hydra/hydra.py]: Fixed.

**Comment by TheDiemer** [comment]
@sferich888 why did you kill needsNewOwner, severity, serviceLevel, and fts in order to add cluster id?

The `__init__`, `__pretty_print_REQ`, and `__get_api` changes you made all LGTM, but you removed functionality in `query_cases`.

**Comment by sferich888** [comment]
@TheDiemer I changed APIs, wert not using the cases API to search or get cases any more were using the search API. It doesn't have the same capabilities.

**Comment by TheDiemer** [comment]
@sferich888 But that removes functionality. I think it would be better to create a secondary function for this capability so we can ADD functionality, not remove things.

I don't see `search by openshift cluster id` as inherently more important than `searching for cases by nno, fts, severity, AND serviceLevel`.

**Comment by sferich888** [comment]
@TheDiemer can you please merge this so that the CLI does not have issues with its query command?

**Comment by TheDiemer** [comment]
As discussed in IRC, the preferred steps forward would be to make the Very minor correction to the existing function so the existing functionality is not removed and make this new endpoint a new function. As this was not done, I took the liberty of writing this up as a new a [new PR](https://github.com/TheDiemer/pyhydra/pull/11) and have merged this one instead. I will now be closing this PR as the requested functionality (search for cases by cluster id) is now included in the tooling.

**Pull request closed without merging**